requirements_pt.txt

```diff
@@ -1,4 +1,4 @@
-auto-round
+auto-round @ git+https://github.com/intel/auto-round.git@eaf3710ca1df0a645acbe97e965f8672ff8ba341
```

Comment:
This requirement file is used by INC3 torch. Please update it in a separate PR if needed.

Reply:
v0.11 has been released, we can remove the version limit.

Reply:
requirements_pt.txt not updated,  so no other PR is needed.

Reply:
removed the version limit https://github.com/intel/neural-compressor/pull/1688/commits/feae5ed7a4d3d49a82ad7c1a6e01cacae07e206c